packages/commons/github/src/ClonedRepository.ts

```diff
@@ -32,16 +34,39 @@ export class ClonedRepository {
     return await this.readFile({ relativeFilePath: README_FILEPATH });
   }
 
+  public async getFernignore(): Promise<string | undefined> {
+    return await this.readFile({ relativeFilePath: FERNIGNORE });
+  }
+
   public async add(files: string | string[]): Promise<void> {
     await this.git.cwd(this.clonePath);
     await this.git.add(files);
   }
 
+  public async restoreStagedFiles(files: string | string[]): Promise<void> {
+    await this.git.cwd(this.clonePath);
+    await this.git.raw([
+      "restore",
+      "--staged",
+      ...(Array.isArray(files) ? files : [files]),
+    ]);
+  }
+
+  public async restoreFiles(files: string[]): Promise<void> {
+    await this.git.cwd(this.clonePath);
+    await this.git.raw([
+      "restore",
+      ...(Array.isArray(files) ? files : [files]),
+    ]);
+  }
```

Comment:
I wonder if we should use an option here and consolidate these functions.

```suggestion
  public async restoreFiles({
    files,
    staged
  }: {
    files: string[],
    staged?: boolean
  }): Promise<void> {
    await this.git.cwd(this.clonePath);
    const args = ["restore"];
    if (staged) {
      args.push("--staged")
    }
    await this.git.raw([
      ...args,
      ...(Array.isArray(files) ? files : [files]),
    ]);
  }
```

That way it's simpler for us to add other options in the future without rolling out separate functions.